src/pages/angular/virtual-scroll.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Virtual Scroll in Angular'
```

Comment:
Should this just be `Virtual Scroll` since we're already in the Angular section?